solidity/dashboard/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_ETH_RPC_URL=$ETH_RPC_URL
```

Comment:
Did you mean to include the `REACT_APP_` prefix? I think `ETH_RPC_URL` would suffice.

Reply:
Yep. Pls check here https://create-react-app.dev/docs/adding-custom-environment-variables/